client/src/App.js

```diff
@@ -1,11 +1,104 @@
+import React, { useState, useEffect } from "react";
 import "./App.css";
+// import SampleData from "./SampleData/exampleresponse.json";
+import ListVideos from "./components/ListVideos";
+import Nav from "./components/Nav";
+import FormDisabled from "./components/FormDisabled";
 
 function App() {
+  const [VideosInfo, setVideosInfo] = useState([]);
+  const [load, isLoaded] = useState(false);
```

Comment:
the convention for state variable names is to have the same name with `set` prepended for the setter, so in this case either

```
const [load, setLoad] = useState(false);
```

or

```
const [isLoaded, setIsLoaded] = useState(false);
```